### PR TITLE
Fix unable to query status endpoint issue.

### PR DESCRIPTION
**Pull request by yunkunrao**
To solve this issue: #96，
I added several rpc related status structures and enable rpc APIs.

### CODE REVIEW

File: tm-monitor/rpc.go [hunk 1]
```diff
@@ -37,23 +42,41 @@ func routes(m *monitor.Monitor) map[string]*rpc.RPCFunc {
...
 
 // RPCNodeStatus returns statistics for the given node.
 func RPCNodeStatus(m *monitor.Monitor) interface{} {
-	return func(name string) (*monitor.Node, error) {
+	return func(name string) (*NodeStatus, error) {
+
 		if i, n := m.NodeByName(name); i != -1 {
-			return n, nil
+
+			status := &NodeStatus{}
```
Comment (melekes) [inline tm-monitor/rpc.go]: there is no need in this additional data structures. you can just add `amino:"unsafe"` tag to the float fields https://github.com/tendermint/tools/pull/98#discussion_r193399934

Comment (yunkunrao) [reply tm-monitor/rpc.go]: Yeah, you are right and I have verified it. So currently could we replace wire:"unsafe" with amino:"unsafe"?

Comment (melekes) [reply tm-monitor/rpc.go]: yes

**Comment by melekes** [comment]
Thank you for working on this!

**Pull request merged**